README.md

```diff
@@ -1,2 +1,81 @@
 # node-axosoft
 A node.js client for the Axosoft API
+
+This library is built to support v6 of the Axosoft REST API and has been tested with both axosoft.com hosted accounts and Axosoft installed on premise.  Any questions can be sent to api@axosoft.com.
+
+[Here's](http://developer.axosoft.com/) a link to our full documentation for the Axosoft REST API.
+
+## Installation
+
+  Install with the node package manager [npm](http://npmjs.org):
+
+    $ npm install node-axosoft
+
+## Examples
+
+### Create the Axosoft Connection
+
+    const nodeAxosoft = require('node-axosoft');
+    const axosoftUrl = {Axosoft Url};
+    var credentials = {};
+
+    //Populate Credentials (See Below)
+
+    var axo = new nodeAxosoft(axosoftUrl, credentials);
+
+### Populate Credentials
+
+  User Name and Password
+
+    credentials.client_id = {client id};
+    credentials.client_secret = {client_secret};
+    credentials.grant_type = 'password';
+    credentials.username = {username};
+    credentials.password = {password};
+
+  Authorization Code (required for public apps or if using Windows authentication)
+
+    credentials.client_id = {client id};
+    credentials.client_secret = {client_secret};
+    credentials.grant_type = 'authorization_code';
+    credentials.redirect_uri = {redirect_uri};
+
+    var authorizationUrl = new nodeAxosoft(axosoftUrl, credentials).Api.getLoginUrl();
+
+    // open browser using authorizationUrl and get code parameter from redirected Url after login
+
+    credentials.code = {code received};
+
+    var axo = new nodeAxosoft(axosoftUrl, credentials);
+
+  Non-Expiring Token
+
+    //Create Non Expiring Token by logging into Axosoft account, Clicking on Tools/System Options/Axosoft API Settings/Manage Tokens, and make non-expiring token.
+    credentials.access_token = {non-expiring token};
+
+    var axo = new nodeAxosoft(axosoftUrl, credentials);
+
+### Get Work Item (feature)
+
+    //optional parameters
+    var params = {columns: 'name'};
+
+    axo.Features.get(params, function(error, item){
+        console.log(item);
```

Comment:
second parameter should be `response`, and the console log should be `response.data` (it's more than 1 item)